internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -405,13 +405,15 @@ func (e *EnvStackConfig) publicHTTPConfig() (template.HTTPConfig, error) {
 		CIDRPrefixListIDs: e.in.CIDRPrefixListIDs,
 		ImportedCertARNs:  e.importPublicCertARNs(),
 		ELBAccessLogs:     elbAccessLogsConfig,
+		SSLPolicy:         e.getPublicSslPolicy(),
```

Comment:
nit (here and elsewhere):
```suggestion
		SSLPolicy:         e.getPublicSSLPolicy(),
```
because https://github.com/golang/go/wiki/CodeReviewComments#initialisms